sdk/state/state_test.go

```diff
@@ -391,6 +414,177 @@ func Test(t *testing.T) {
 	assert.EqualValues(t, initiatorEscrowResponse.Balances[0].Balance, fmt.Sprintf("%.7f", float64(iBalanceCheck)/float64(1_000_0000)))
 }
 
+func TestOpenUpdatesCoordinatedClose(t *testing.T) {
+	initiator, responder := initAccounts(t)
+	initiatorChannel, responderChannel := initChannels(t, initiator, responder)
+
+	s := initiator.EscrowSequenceNumber + 1
+	i := int64(1)
+	e := int64(0)
+	t.Log("Vars: s:", s, "i:", i, "e:", e)
+
+	// Open
+	t.Log("Open...")
+	open, err := initiatorChannel.ProposeOpen()
+	require.NoError(t, err)
+	for {
+		var fullySignedR bool
+		open, fullySignedR, err = responderChannel.ConfirmOpen(open)
+		if err != nil {
+			t.Fatal(err)
+		}
+		var fullySignedI bool
+		open, fullySignedI, err = initiatorChannel.ConfirmOpen(open)
+		if err != nil {
+			t.Fatal(err)
+		}
+		if fullySignedI && fullySignedR {
+			break
+		}
+	}
+
+	{
+		ci, di, fi, err := initiatorChannel.OpenTxs()
+		require.NoError(t, err)
+
+		ci, err = ci.AddSignatureDecorated(open.CloseSignatures...)
+		require.NoError(t, err)
+
+		di, err = di.AddSignatureDecorated(open.DeclarationSignatures...)
+		require.NoError(t, err)
+
+		fi, err = fi.AddSignatureDecorated(open.FormationSignatures...)
+		require.NoError(t, err)
+
+		fbtx, err := txnbuild.NewFeeBumpTransaction(txnbuild.FeeBumpTransactionParams{
+			Inner:      fi,
+			FeeAccount: initiator.KP.Address(),
+			BaseFee:    txnbuild.MinBaseFee,
+		})
+		require.NoError(t, err)
+		fbtx, err = fbtx.Sign(networkPassphrase, initiator.KP)
+		require.NoError(t, err)
+		_, err = client.SubmitFeeBumpTransaction(fbtx)
+		require.NoError(t, err)
+	}
+
+	// Perform a number of iterations, much like two participants may.
+	// Exchange signed C_i and D_i for each
+	t.Log("Subsequent agreements...")
+	rBalanceCheck := responder.Contribution
+	iBalanceCheck := initiator.Contribution
+	for i < 20 {
+		i++
+		require.Equal(t, i, initiatorChannel.NextIterationNumber())
+		require.Equal(t, i, responderChannel.NextIterationNumber())
+		amount := randomPositiveInt64(t, 100_0000000)
```

Comment:
similarly here too, can we put the number in a named variable. helps with readability

Reply:
added a comment, I think that should be clearer 😄 
https://github.com/stellar/experimental-payment-channels/pull/83/commits/d48f3be031f4512fbed2f78979b6259366b4a724